fabfile.py

```diff
@@ -92,6 +92,7 @@ def decrypt_secrets(c):
 def install_deps(c: Connection):
     print('-- Installing dependencies')
     with c.cd(c.release_path):
+        # TODO: Edit out pipenv dependency
         c.run('source .venv/bin/activate && pipenv install --deploy', echo=True, env={'PIPENV_VENV_IN_PROJECT': 'true'})
```

Comment:
Pretty sure this ought to be `.venv/bin/pip install -r requirements.txt`. The intent here is that if already in an activated virtualenv, pipenv will install into the shared virtualenv which `.venv` symlinks to `hknweb/prod/shared/venv`, so if you can re-leverage that it should still work.

Reply:
Replaced with the init logic similar to our `Makefile` and in `travis.yml`